### DISCUSSION TITLE
Can I use Rustdesk for free at my workplace for commercial purposes?

### DISCUSSION THREAD

**Discussion by RemixPL1994** [rustdesk/rustdesk#9320] [General]
Hello. Can I use Rustdesk for free at my workplace for commercial purposes?

Is it free for home/individual use or is it also free if, for example, I use Rustdesk once a month to help someone remotely for a fee for which I issue a receipt/invoice for this event?

**Comment by basilgello** [comment]
The license of the project is AGPL-3.0 which is not entirely free as the freedom (it restricts cloud providers from parasiting on opensource development) but if you are not a big cloud provider like Amazon or Goohle, you are fine.
